e2e/tests/functional/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -315,9 +296,45 @@ test.describe('Example Imagery in Display Layout', () => {
     await page.locator('div[title="Resize object height"] > input').click();
     await page.locator('div[title="Resize object height"] > input').fill('100');
 
-    expect(thumbsWrapperLocator.isVisible()).toBeTruthy();
-    await expect(thumbsWrapperLocator).not.toHaveClass(/is-small-thumbs/);
-  });
+        expect(thumbsWrapperLocator.isVisible()).toBeTruthy();
+        await expect(thumbsWrapperLocator).not.toHaveClass(/is-small-thumbs/);
+    });
+
+    /**
+     * Toggle layer visibility checkbox by clicking on checkbox label
+     * - should toggle checkbox and layer visibility for that image view
+     * - should NOT toggle checkbox and layer visibity for the first image view in display
+     */
+    test('Toggle layer visibility by clicking on label', async ({ page }) => {
+        await createImageryView(page);
+        await page.goto(displayLayout.url);
+
+        const imageElements = await page.locator('.c-imagery__main-image-wrapper');
+
+        await expect(imageElements).toHaveCount(2);
+
+        const imageOne = await page.locator('.c-imagery__main-image-wrapper').nth(0);
```

Comment:
Nice work with these locators! Small nitpick, `.locator()` is synchronous so we can remove the `await` from each of these calls.